src/Gizmos/gizmo.ts

```diff
@@ -161,7 +163,33 @@ export class Gizmo implements IDisposable {
         if (!this._attachedNode) {
             return;
         }
-        if (this._attachedNode.getClassName() === "Mesh" || this._attachedNode.getClassName() === "AbstractMesh" || this._attachedNode.getClassName() === "TransformNode") {
+        var inheritsTargetCamera = this._attachedNode.getClassName() === "FreeCamera"
+                        || this._attachedNode.getClassName() === "FlyCamera"
+                        || this._attachedNode.getClassName() === "ArcFollowCamera"
+                        || this._attachedNode.getClassName() === "TargetCamera"
+                        || this._attachedNode.getClassName() === "TouchCamera"
+                        || this._attachedNode.getClassName() === "UniversalCamera";
+        if  (this._attachedNode.getClassName() === "ArcRotateCamera" || inheritsTargetCamera) {
+            var camera = this._attachedNode as Camera;
+            var transformQuaternion = new Quaternion(0, 0, 0, 1);

```

Comment:
Same as above - if this is constantly called, you can keep the GC from over-working if you use the predefined Temp vectors/quaternion/matrix object(s)